batch/batch/worker/worker.py

```diff
@@ -1812,6 +2137,22 @@ def __init__(self, client_session: httpx.ClientSession):
         self.headers = None
         self.compute_client = None
 
+        self.jvm_initializer_task = asyncio.ensure_future(self._initialize_jvms())
+        self.jvms: List[JVM] = []
+
+    async def _initialize_jvms(self):
+        self.jvms = await asyncio.gather(*[JVM.create(i) for i in range(CORES)])
```

Comment:
Should this be `4 * CORES`? If not, should JVM Jobs have a minimum CPU of 1?

Reply:
Ah, JVM jobs are always 1 core no matter what. I'll add to todo list to verify that in front end.